webhooks/pod_webhook.go

```diff
@@ -230,6 +252,19 @@ func (m *PodMutator) injectSidecar(
 		}
 	}
 
+	// append raw sidecar container args
+	if flagSourceConfig.RawSidecarArgs != nil {
+		sidecar.Args = append(sidecar.Args, flagSourceConfig.RawSidecarArgs...)
+	}
```

Comment:
I would have raw argument either as the first thing or last thing added to the array. I suggest adding them with the least priority over existing arguments. My motivation is that if I configured a flag directly, eg `debugLogging=true`, and also via `rawSidecarArgs`, the direct value should win to avoid confusion.

Reply:
let's continue the discussion here: https://github.com/open-feature/open-feature-operator/pull/434#discussion_r1158042848